app/src/main/java/org/piramalswasthya/sakhi/ui/home_activity/all_ben/new_ben_registration/ben_form/NewBenRegViewModel.kt

```
@@ -248,9 +266,60 @@ class NewBenRegViewModel @Inject constructor(
         _recordExists.value = b
     }
 
+     fun sentOtp(mobileNo: String) {
+        viewModelScope.launch {
+            benRepo.sendOtp(mobileNo)?.let {
+                if (it.status == "Success") {
+
+                } else {
+
+
+                }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyElseBlock)